client/src/components/Draftail/Draftail.scss

```
@@ -191,7 +191,7 @@ $draftail-editor-font-family: $font-sans;
 }
 
 .Draftail-ComboBox__menu {
-  padding: 0 theme('spacing.4');
+  padding: theme('spacing.4') theme('spacing.4');
```

```suggestion
  padding: theme('spacing.4');
```

If the values are the same here we can use the shorthand version, I will apply this change quickly now.